### DISCUSSION TITLE
Some questions about the project.

### DISCUSSION THREAD

**Discussion by maomingyang1314** [Chia-Network/chia-blockchain#4372] [Support]
Dear Huffman, I would like to ask whether the 1.1.5 update will cause the previous version of the drawing to become invalid. I watched your meeting on YoTube, and it looks like you need to update the agreement. If the previous drawing becomes invalid in the future, it will be destructive. @hoffmang9

**Comment by maomingyang1314** [comment]
The official mining pool, do we need to redraw the drawing? I remember that you said at the web conference that the previous drawing can be used and you can directly access the mining pool

**Comment by maomingyang1314** [comment]
Recently in China, there have been many comments that we need to redraw the picture. The disk space capacity of the entire network has also dropped today

**Reply by SackofFluor** [reply]
Yes, I am also still looking for an explaination why the Network dropped in the last 24h.

I think the official pool protocol is starting in a week or so

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
